src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,33 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Association Class 
+ * Adds and removes tags to a person in the address book.
+ */
+public class Tagging {
+    
+    Person person;
+    Tag tag;
+    boolean isValid;
```

Comment:
A boolean variable works here, but an `enum` may be more appropriate because the two values we need are `added` and `deleted`, not `true` and `false`.